Objects/object_layout.md

```diff
@@ -62,7 +62,7 @@ the values pointer, to enable the (legacy) C-API function
 * ob_refcnt
 * ob_type
 
-For a "normal" Python object, that is one that doesn't inherit from a builtin
+For a "normal" Python object, i.e., the one that doesn't inherit from a builtin
```

Comment:
'the one' is not correct, and a bit worse than previously.  Following is better.
```suggestion
For a "normal" Python object, one that doesn't inherit from a builtin
```

Reply:
Yeah, that sounds much better. Updated.